### ISSUE TITLE
Create guided-install for CCF

### ISSUE THREAD

**Issue by ericksod** [cloud-carbon-footprint/cloud-carbon-footprint#134]
#### Overview
We would like to remove as many barriers as possible for folks trying to get set up. As part of this, we were inspired by the create-app package that Backstage has. This would be a package that creates a whole new repo and all the source files to allow for customization.
We considered that approach and decided to start off with a slightly less involved approach. The user will clone the latest release of the repo, and then a script will guide them through setting up the necessary configurations for the cloud provider(s) they want to connect to.
#### Out of Scope:
- keeping main branch stable or some other latest release solution
- instructions to clone repo with latest release
- full create-app package which creates new repo
## Acceptance Criteria
- [x] When I clone the latest release of the repo, I can run a command `yarn guided-install` (or other phrasing as you see fit)
- [x] When I run the guided install command, I see a welcome message, for example `Please follow the following prompts to configure Cloud Carbon Footprint to connect to a cloud provider. To connect to multiple cloud providers, please repeat this script for each additional provider. For an understanding of the default configurations and other configuration options, please visit the docs on the microsite: https://www.cloud-carbon-footprint.org/docs/running-locally` <- not sure yet if this will be url
- [x] When I have finished reading the welcome message, I am prompted to select which cloud provider I would like to configure
- [x] ? When I have selected a valid cloud provider, I am given the set up instructions specific to that cloud provider? I am linked to the docs page? eg. setting up Athena DB to save Cost and Usage reports or getting Google Cloud billing data to export to Big Query [https://redesigned-guide-07c796a7.pages.github.io/docs/aws](https://redesigned-guide-07c796a7.pages.github.io/docs/aws)
- [x] When I go through the guided-install, the option `USE_BILLING_DATA` for my cloud provider should be set to true by default
- [x] When I have selected a cloud provider and have prepared that account to have data pulled, I am prompted to enter the necessary env variables for the Billing Data approach, eg `GCP_BIG_QUERY_TABLE`, `GCP_BILLING_ACCOUNT_ID`, `GCP_BILLING_ACCOUNT_NAME`, `GCP_PROJECTS`
- [x] When I have completed the guided-install, I see a completion message, for example `Your Cloud Carbon Footprint application is now connected to your cloud provider data. You can see these settings in [path-to-env-file] and [path-to-config-file]. To connect to an additional cloud provider or update an existing one, please re-run this script. For questions or more configuration options, please visit [link-to-microsite]`
- [x] When I have completed the guided-install, I can see an env file with my cloud provider variables set
- [ ] When I have completed the guided-install, the cloud provider I added is part of the Cloud Providers array in my config file
- [x] When I run and do not complete the guided-install, it does not make updates in the env or config files
- [x] When I run and complete the guided-install twice for different cloud providers, I can see both configured in my env file and cloud providers array in the config file
- [x] When I run and complete the guided-install twice for the same cloud provider, it updates the env variables
- [x] When I run and complete the guided-install twice for the same cloud provider, I see the cloud provider only once in the config file

## Tech Tasks
- [x] The guided-install script lives in the CLI package
- [x] The guided-install script is written in Typescript
